src/common/helpers/DeclarationIndexHelpers.ts

```diff
@@ -160,7 +160,7 @@ export async function findFiles(config: ExtensionConfig, workspaceFolder: Worksp
         o.filter(
             f => f.fsPath
                 .replace(rootPath || '', '')
-                .split(/\\|\//)
+                .split(/[\\/]/)
```

Comment:
Alternatives are costly, classes are preferable for single chars (you've done that most elsewhere, btw)